wpimath/src/main/java/edu/wpi/first/math/jni/Twist3dJNI.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.math.jni;
+
+/** Twist3d JNI. */
+public final class Twist3dJNI extends WPIMathJNI {
+  /**
+   * Obtain a Transform3d from a (constant curvature) velocity.
+   *
+   * <p>The double array returned is of the form [dx, dy, dz, qx, qy, qz].
```

Comment:
```suggestion
   * <p>The double array returned is of the form [dx, dy, dz, qw, qx, qy, qz].
```
My bad for this- I got the comment wrong when I first wrote it. Still, we might as well fix this while we're at it.